rfcs/050-concepts-api/README.md

```diff
@@ -0,0 +1,194 @@
+# RFC 050: Design considerations for the concepts API
+
+This RFC collects some initial thinking on how we might represent concepts in the catalogue API.
+It's a starting point for discussions; not a final design.
+
+
+
+## Goals
+
+*   We want to be able to create per-person and per-subject pages on the website.
+    The data on these pages should come from the new concepts API.
+
+*   These pages should contain enough context to distinguish two people/subjects (e.g. whether "Charles Darwin" is the famous naturalist, his uncle, his son, or his grandson).
+
+*   We're not trying to replicate Wikidata -- the APIs shouldn't return large amounts of information we aren't going to use on wellcomecollection.org.
+
+*   Any new APIs should be similar to the existing works and images APIs.
+    We'll use the same conventions for filters, aggregations, including optional fields, and so on.
+
+
+
+## Endpoint #1: retrieve a single concept
+
+This is the single concept endpoint, which is used to render a page for an individual subject or person:
+
+```
+GET /concepts/azxzhnuh
+{
+  "id": "azxzhnuh",
+  "identifiers": [
+    {
+      "identifierType": "lc-names",
+      "value": "n12345678",
+      "type": "Identifier"
+    },
+    ...
+  ],
+
+  "label": "Florence Nightingale",
+  "alternativeLabels": [
+    "The Lady with the Lamp",
+  ]
+
+  "type": "Person|Subject|Organisation|Place"
+
+  // Everything above here is stuff I'm pretty sure we'll need;
+  // everything below it is more nebulous and more likely to change
+  // in the final API.
+
+  "description": "Florence Nightingale was an English social reformer, statistician and the founder of modern nursing. Nightingale came to prominence while serving as a manager and trainer of nurses during the Crimean War, in which she organised care for wounded soldiers at Constantinople.",
+
+  // not locations
+  "urls": [
```

Comment:
Are Uniform Resource Locators really not locations?

Can we find a way of describing what these are _in relation to the concept_? Florence Nightingale didn't have any URLs (she had a photographic memory for IP addresses).

I think these might want to be enriched with some description of what they are - if I want to render a works page with a link to Wikipedia, I'll want to know which of the `urls` is the wikipedia link. 

Starting point (not a final suggestion) to address my points above; inspiration taken from Wikidata:
```json
{
  "describedAt": [
    {
      "source": "wikipedia"
      "url": "https://en.wikipedia.org/wiki/Florence_Nightingale"
    }
  ]
}
```

Reply:
But I do think there's something to the idea of `describedAt` as a way to say "learn more about this concept over here".

Reply:
Yeah, leaning towards `DigitalLocation`s being OK - if we take one thing from this comment then it's that I don't really like `urls` as a name and I do rather like `describedAt` - cf https://www.wikidata.org/wiki/Property:P973

Reply:
I used a Wikipedia URL as an example; I went for the slightly more generic `urls` because I wasn't sure if we'd ever want other links here (e.g. will every concept have a Wikipedia entry, or might we link elsewhere for some things?)